cpp/tests/strings/reverse_tests.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cudf_test/base_fixture.hpp>
+#include <cudf_test/column_utilities.hpp>
+#include <cudf_test/column_wrapper.hpp>
+
+#include <cudf/column/column.hpp>
+#include <cudf/strings/reverse.hpp>
+#include <cudf/strings/strings_column_view.hpp>
+
+#include <vector>
+
+struct StringsReverseTest : public cudf::test::BaseFixture {
+};
+
+TEST_F(StringsReverseTest, Reverse)
+{
+  cudf::test::strings_column_wrapper input({"abcdef", "12345", "", "", "aébé", "A é Z"});
```

Comment:
Can we include a null value in some test?

Reply:
Yes, I have more tests coming. This is a draft PR.